scripts/apps/authoring/metadata/views/metadata-dropdown.html

```diff
@@ -3,10 +3,14 @@
         {{ key && values[item[field]]  && values[item[field]].name ? values[item[field]].name: (item[field].name || item[field] ? item[field].name || item[field] : "none") | translate }}
         <b class="dropdown__caret"></b>
     </button>
-    <button ng-disabled="disabled" class="dropdown__toggle line-input" tabindex="{{tabindex}}" dropdown__toggle ng-if="listFields.indexOf(field) > -1 && !icon">
+    <button ng-disabled="disabled" class="dropdown__toggle line-input" tabindex="{{tabindex}}" dropdown__toggle ng-if="listFields.indexOf(field) > -1 && !icon && !cv">
```

Comment:
It would be useful to comment under which conditions which button is displayed since that's hard to understand.